src/components/Layout/SymptomsCards/SymptomsCards.scss

```diff
@@ -0,0 +1,47 @@
+.Carrousel{
+  width: 100%;
+  overflow: scroll;
+  padding: 20px;
+  position: relative; 
+}
+.Carrousel__title {
+  font-size: 16px;
+  color: #929292;
+  padding: 0px 10px;
+}
```

Comment:
Hello @Sergioleal7 
I think here you can use this feature of Sass: [Adding Suffixes](https://sass-lang.com/documentation/style-rules/parent-selector#adding-suffixes) to avoid repeat some words and take advantage of nesting feature of Sass.

Reply:
Should be like this
```
.Carrousel{
  .....
&__title {
  .....
  }
}
```
.Can apply to the other classes